jsconfig.json

```diff
@@ -3,11 +3,12 @@
     "baseUrl": ".",
     "sourceMap": true,
     "paths": {
-      "common-util": ["./common-util"],
-      "components": ["./components"],
-      "util": ["./util"],
-      "static": ["./static"],
-      "images": ["./public/images"]
+      "common-util/*": ["./common-util/*"],
+      "components/*": ["./components/*"],
+      "util/*": ["./util/*"],
```

Comment:
path requires a star on the end to work with build, otherwise it only applies to top level folder